question/qbanksettings.php

```diff
@@ -0,0 +1,57 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Question bank settings page class.
+ *
+ * @package    qbank_settingspage
+ * @author     Ghaly Marc-Alexandre <marc-alexandreghaly@catalyst-ca.net>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+use core_question\local\bank\helper;
+
+require_once(__DIR__ . '/../config.php');
+require_once($CFG->libdir.'/adminlib.php');
+
+require_login();
+$section = required_param('section', PARAM_SAFEDIR);
+
+$PAGE->set_context(context_system::instance());
+$PAGE->set_url('/question/qbanksettings.php', array('section' => $section));
+$PAGE->set_title(get_string($section, 'admin'));
+$PAGE->set_heading($SITE->fullname);
+$PAGE->set_pagelayout('admin');
+$PAGE->requires->js_call_amd('core_question/drag_drop', 'init');
+$PAGE->navigation->clear_cache();
+
+$context = array();
+
+$corequestionbankcolumns = helper::get_question_list_columns();
+foreach ($corequestionbankcolumns as $columnname) {
+    $name[] = $columnname->name . ' {' . $columnname->colname . '}';
+    $names['names'] = $name;
+}
+
+$context = $names;
+$context['header'] = get_string($section, 'admin');
```

Comment:
No need to send a header, as the title should be done by the $PAGE->set_heading(...

Reply:
That was forgotten here, removed, thanks !